src/Libraries/Analysis/VectorData.cs

```diff
@@ -87,15 +87,15 @@ private void DrawVector(Tuple<Vector, Point> data, IRenderPackage package)
             var p = data.Item2;
             var v = data.Item1;
 
-            package.PushLineStripVertex(p.X, p.Y, p.Z);
-            package.PushLineStripVertexColor(120, 120, 120, 255);
+            package.AddLineStripVertex(p.X, p.Y, p.Z);
+            package.AddLineStripVertexColor(120, 120, 120, 255);
 
             var o = p.Add(v);
 
-            package.PushLineStripVertex(o.X, o.Y, o.Z);
-            package.PushLineStripVertexColor(120, 120, 120, 255);
+            package.AddLineStripVertex(o.X, o.Y, o.Z);
+            package.AddLineStripVertexColor(120, 120, 120, 255);
 
```

Comment:
When you're repeatedly using the same number over and over, you know I'm going to say it should be `const`.  :)